cordova-common/src/CordovaLogger.js

```diff
@@ -209,7 +209,7 @@ function formatError(error, isVerbose) {
         message = error;
     }
 
-    if(message.toUpperCase().indexOf('ERROR:') !== 0) {
+    if(message && typeof message.toUpperCase === 'function' && message.toUpperCase().indexOf('ERROR:') !== 0) {

```

Comment:
This is fine, but I would have preferred `if (typeof message === 'string' && message.toUpperCase()...)`